local/langdata_127/default.php

```diff
@@ -617,7 +619,9 @@
 		"INCLUDE_PATIENT_NAME" => "Include patient name",
 		"INCLUDE_PATIENT_BIRTHDATE" => "Include patient birth date",
 		"INCLUDE_PATIENT_SEX" => "Include patient sex",
-        "INCLUDE_PATIENT_ID" => "Include patient ID"
```

Comment:
Can you also update _all_ the XML files where you changed the PHP files? (You can copy paste)

Recent experience on my end shows that we should keep local/langdata_127, langdata_12, and langdata_revamp in sync.

Reply:
Updated